<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaManDialog</class>
 <widget class="QDialog" name="MediaManDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>对讲管理界面</string>
  </property>
  <widget class="QTableView" name="tableViewMediaList">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>77</y>
     <width>518</width>
     <height>377</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaManReturn">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>400</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaManNext">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>144</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaManDel">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>272</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaManPrev">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>80</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaManDetail">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>208</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>详细</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMediaManDelAll">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>336</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>全删</string>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>18</y>
     <width>538</width>
     <height>450</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>20</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>,white</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>对讲管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>framePanel</zorder>
  <zorder>tableViewMediaList</zorder>
  <zorder>pushButtonMediaManReturn</zorder>
  <zorder>pushButtonMediaManNext</zorder>
  <zorder>pushButtonMediaManDel</zorder>
  <zorder>pushButtonMediaManPrev</zorder>
  <zorder>pushButtonMediaManDetail</zorder>
  <zorder>pushButtonMediaManDelAll</zorder>
  <zorder>label_title</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonMediaManPrev</tabstop>
  <tabstop>pushButtonMediaManNext</tabstop>
  <tabstop>pushButtonMediaManDetail</tabstop>
  <tabstop>pushButtonMediaManDel</tabstop>
  <tabstop>pushButtonMediaManDelAll</tabstop>
  <tabstop>pushButtonMediaManReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
